plugins/SlicerT/SlicerT.h

```diff
@@ -0,0 +1,142 @@
+/*
+ * SlicerT.h - declaration of class SlicerT
+ *
+ * Copyright (c) 2023 Daniel Kauss Serna <daniel.kauss.serna@gmail.com>
+ *
+ * This file is part of LMMS - https://lmms.io
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program (see COPYING); if not, write to the
+ * Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301 USA.
+ *
+ */
+
+#ifndef LMMS_SLICERT_H
+#define LMMS_SLICERT_H
+
+#include <fftw3.h>
+
+#include "AutomatableModel.h"
+#include "Instrument.h"
+#include "InstrumentView.h"
+#include "Note.h"
+#include "PhaseVocoder.h"
+#include "SampleBuffer.h"
+#include "SlicerTView.h"
+
+namespace lmms {
+
+// simple helper class that handles the different audio channels
+class DynamicPlaybackBuffer
+{
+public:
+	DynamicPlaybackBuffer()
+		: m_leftChannel()
+		, m_rightChannel()
+	{
+	}
+
+	void loadSample(const sampleFrame* outData, int frames, int sampleRate, float newRatio)
```

Comment:
The `outData` parameter should probably be renamed `inData`

Reply:
Done